### PR TITLE
De-robusts simple animal attacks

### PR DESCRIPTION
**Pull request by LorenLuke**
Adds small delay to simple_animal attack logic, and gives their attacks a miss probability.

### CODE REVIEW

**Comment by Anewbe** [comment]
Start with 25, not 40, please.

**Comment by LorenLuke** [comment]
@Anewbe 
>25 not 40

I had, originally. Bumped up after testing with various weapons, and still getting wrecked.

Checked something, was going up against 200 health spiders only. Dropping back to 25%.

File: code/modules/mob/living/simple_animal/simple_animal.dm [hunk 1]
```diff
@@ -1117,9 +1118,18 @@
...
 	if(!Adjacent(target_mob))
 		return
+	sleep(rand(8) + 8)
 	if(isliving(target_mob))
 		var/mob/living/L = target_mob
-		L.attack_generic(src,rand(melee_damage_lower,melee_damage_upper),attacktext)
+
+		if(prob(melee_miss_chance))
+			src.attack_log += text("\[[time_stamp()]\] <font color='red'>attacked [L.name] ([L.ckey])</font>")
+			L.attack_log += text("\[[time_stamp()]\] <font color='orange'>was attacked by [src.name] ([src.ckey])</font>")
```
Comment (Anewbe) [inline code/modules/mob/living/simple_animal/simple_animal.dm]: Why are you logging the misses?

Comment (LorenLuke) [inline code/modules/mob/living/simple_animal/simple_animal.dm]: Regular unarmed attacks for players, iirc, log misses, so included here for sake of completeness.

**Pull request merged**